src/data/apiClient.js

```diff
@@ -9,6 +10,7 @@ const apiClient = getAuthenticatedAPIClient({
   refreshAccessTokenEndpoint: process.env.REFRESH_ACCESS_TOKEN_ENDPOINT,
   accessTokenCookieName: process.env.ACCESS_TOKEN_COOKIE_NAME,
   userInfoCookieName: process.env.USER_INFO_COOKIE_NAME,
+  loggingService: NewRelicLoggingService,
```

Comment:
This is required as of version `5.0.0` of edx/frontend-auth. We are now on `5.3.1`